src/main/drivers/dshot.h

```diff
@@ -103,7 +103,7 @@ typedef struct dshotTelemetryState_s {
     uint32_t readCount;
     dshotTelemetryMotorState_t motorState[MAX_SUPPORTED_MOTORS];
     uint32_t inputBuffer[MAX_GCR_EDGES];
-    uint32_t averageRpm;
+    uint16_t averageErpm;
```

Comment:
Is it guaranteed that this value cannot overflow? What's the max. possible average eRPM value?

Reply:
Yes, because the erpm value is coded in dshot as eeem.mmmm.mmmm. Being mmmmmmmmm (0-511) the base and eee (0-7) the shift, so the maximum value that can be coded is 511 << 7 = 65408, some values above that value to 65535 are used as error ones in the code.